packages/content-conformance/package.json

```diff
@@ -19,7 +19,10 @@
     "unist-util-visit": "^4.1.1",
     "vfile": "^5.3.5",
     "vfile-reporter": "^7.0.4",
-    "yaml": "^2.1.3",
```

Comment:
We still want this dependency, it looks like this might have been a merge conflict casualty.

Reply:
Added it back!